lineman/app/components/notification_volume/notification_volume_dropdown.haml

```diff
@@ -1,4 +1,4 @@
-.notification-volume{dropdown: ''}
+.notification-volume{ng-show: 'canChangeVolume()', dropdown: ''}
```

Comment:
Let's make these ng-if, since they're unlikely to change during the course of a page visit.